documentation/src/main/asciidoc/topics/proc_xsite_monitoring.adoc

```diff
@@ -0,0 +1,69 @@
+[id='monitor-xsite-replication_{context}']
+= Monitoring the Cross-Site Replication
+
+After enabling cross-site replication with your {brandname} server, the replication status should be monitored to detect interruptions in the communication between sites.
+When a remote site changes status to `offline`, {brandname} stops replicating your data to that specific site.
+Your data becomes out of sync between your sites.
+
+Monitoring these events is necessary.
+Admin intervention is required to fix the inconsistencies in your data and to re-enable cross-site replication.
```

Comment:
Hi @pruivo I'm on it. What do you mean here by admin intervention?
I'm thinking about something like: "Your data become out of sync, you need to bring the clusters back online and preform state transfer operations."